### DISCUSSION TITLE
Verion number / update check

### DISCUSSION THREAD

**Discussion by ddkooker2** [docmost/docmost#224] [Help / Questions] [answered]
I have been searching everywhere and can't seem to find it, is there somewhere a version number listed in the webui to doublecheck the build that i'm currently running?

**Comment by alohl669** [comment]
If you are using it less than 22 days, you probably are running the latest version 0.2.10. I pulled the latest docker version a few days ago following their installation guide and I had the same question. I just compared the images ID in the docker "tags" section.

**Comment by fengdu126** [comment, marked as answer]
You can refer to the diagram below
![image](https://github.com/user-attachments/assets/bda0ceef-1dca-4254-a0f4-e87f9281f460)
